### DISCUSSION TITLE
Support for Smart Heat Pump

### DISCUSSION THREAD

**Discussion by Kr0n1c** [tuya/tuya-homebridge#400] [General]
Have a heat pump hot water - added as a smart heat pump in the app.

The device successfully is added to homebridge and HomeKit, but is reported as an unsupported device.

This is also in the logs: “ [Tuya] Unsupported device: Heat Pump.”. 

Is it possible to change the device type to a thermostat/AC - allowing the following controls:
- on/off
- Current water temperature
- Temperature control

**Comment by jcbmic012** [comment]
Did you ever get to a solution or workaround on this?
